docker/rootfs/etc/services.d/go2rtc/run

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+if [ -f "/config/frigate-go2rtc.yaml" ]; then
+    /usr/local/go2rtc/sbin/go2rtc -config=/config/frigate-go2rtc.yaml
```

Comment:
```suggestion
    exec /usr/local/go2rtc/sbin/go2rtc -config=/config/frigate-go2rtc.yaml
```

You have to add exec, otherwise Bash will keep running.